### PR TITLE
Check the Cython and Python version in setup.py

### PR DESCRIPTION
**Pull request by Sumith1896**
Fixes https://github.com/symengine/symengine/issues/93
cc @isuruf

### CODE REVIEW

**Comment by isuruf** [comment]
Setting cython with a version number in `setup_requires` would check the cython version right?

**Comment by Sumith1896** [comment]
You mean `setup_requires = ['cython>=0.19.1'] ` ?

**Comment by certik** [comment]
Yes, I think we can use setuptools for the versioning, instead of writing our own code. Plus, we check this in cmake anyway.

**Comment by Sumith1896** [comment]
Agreed, will update

**Comment by Sumith1896** [comment]
Build seems to fail for `Python 3X` tests, any work around?

File: setup.py [hunk 1]
```diff
@@ -173,6 +173,7 @@ def run(self):
 setup(name = "symengine",
       version = "0.1.0.dev",
       description = "Python library providing wrappers to SymEngine",
+      setup_requires = ['cython>=0.19.1', 'python>=2.6'],
```
Comment (isuruf) [inline setup.py]: You can enforce the python version as before and remove it from here.

Comment (Sumith1896) [inline setup.py]: Will do.

**Comment by isuruf** [comment]
+1

**Comment by certik** [comment]
Thanks @Sumith1896 !

**Comment by Sumith1896** [comment]
My pleasure :)

**Pull request merged**